packages/features/eventtypes/lib/getPublicEvent.ts

```diff
@@ -141,24 +146,25 @@ export const getPublicEvent = async (username: string, eventSlug: string, prisma
     };
   }
 
+  const usersOrTeamQuery = isTeamEvent
+    ? {
+        team: {
+          slug: username,
+        },
+      }
+    : {
+        users: {
+          some: {
+            username,
+          },
+        },
```

Comment:
```suggestion
        },
        team: null,
        
```

Reply:
This is needed, otherwise it would also return team events as well and then again we can serve team event instead of user only events.

Reply:
Wow, this makes a lot of sense yeah! Good spot!